src/index.html

```diff
@@ -1,13 +1,291 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Nothing</title>
```

Comment:
Change the title tag content to be more descriptive and relevant to the landing page.